7.2/Dockerfile

```diff
@@ -88,4 +88,4 @@ RUN sudo php -r "copy('https://getcomposer.org/installer', 'composer-setup.php')
 	sudo php composer-setup.php --version=$COMPOSER_VERSION --install-dir=/usr/local/bin --filename=composer && \
 	sudo php -r "unlink('composer-setup.php');" && \
 	composer --version
-ENV PATH /home/circleci/.config/composer/vendor/bin:/home/circleci/.composer/vendor/bin:$PATH
+ENV PATH /home/circleci/.config/composer/vendor/bin:/home/circleci/.composer/vendor/bin:$PATH
```

Comment:
Can you fix your editor to stop doing this?

Reply:
uhh yea let me look into what's causing this, unless you know what's going on here